README.rst

```diff
@@ -183,8 +183,20 @@ click-odoo-makepot (stable)
                                 Merge .pot changes into all .po files, only
                                 if a new .pot file has been created.
                                 [default: False]
+    --fuzzy-matching /  --no-fuzzy-matching
+                                Use fuzzy matching when merging .pot changes
+                                into .po files.
+                                Only applies when --msgmerge
+                                or --msgmerge-if-new-pot are passed.
+                                [default: True]
     --commit / --no-commit      Git commit exported .pot files if needed.
                                 [default: False]
+    --purge-old-translations /  --no-purge-old-translations
+                                Remove comment lines containing old
+                                translations from .po files.
+                                Only applies when --msgmerge
+                                or --msgmerge-if-new-pot are passed.
+                                [default: False]
```

Comment:
A small detail, but it would be nicer if the new options were listed close together with the `--msgmerge` options. `--commit` in the middle looks a bit weird.

Reply:
Thanks for the suggestion! :+1: